disnake/interactions/application_command.py

```diff
@@ -127,6 +127,14 @@ class ApplicationCommandInteraction(Interaction[ClientT]):
 
         .. versionadded:: 2.10
 
+    attachment_size_limit: :class:`int`
+        The maximum number of bytes files can have in responses to this interaction.
```

Comment:
```suggestion
        The maximum number of :class:`bytes` files can have in responses to this interaction.
```
Kind of optional, I'm not sure if it's necessary

Reply:
This refers more to actual bytes, not the python `bytes` object in particular, so I'd say a plain "bytes" works here. `Guild.filesize_limit` is similar:
https://github.com/DisnakeDev/disnake/blob/bc7e9edd00db8573219c950038dd6de0922f9b14/disnake/guild.py#L941-L942

Reply:
oh, right!